File: comguanmuonehours/src/main/java/com/guanmu/onehours/activity/AboutActivity.java
package com.guanmu.onehours.activity;

import android.os.Bundle;
import androidx.appcompat.widget.Toolbar;
import android.view.View;

import com.guanmu.onehours.R;

public class AboutActivity extends BaseActivity {

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.about_activity);

		Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
		setSupportActionBar(toolbar);
		getSupportActionBar().setDisplayHomeAsUpEnabled(true);
		toolbar.setNavigationOnClickListener(new View.OnClickListener() {
			@Override
			public void onClick(View v) {
				finish();
			}
		});

	}
}

File: comguanmuonehours/src/main/java/com/guanmu/onehours/activity/AboutActivity.kt
package com.guanmu.onehours.activity

import android.os.Bundle
import androidx.appcompat.widget.Toolbar
import android.view.View

import com.guanmu.onehours.R

class AboutActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.about_activity)

        val toolbar = findViewById<View>(R.id.toolbar) as Toolbar
        setSupportActionBar(toolbar)
        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
        toolbar.setNavigationOnClickListener { finish() }

    }
}
